### DISCUSSION TITLE
Add DCTERMS metadata to pages?

### DISCUSSION THREAD

**Discussion by ThrawnCA** [ckan/ckan#6855] [Ideas]
Some of the Dublin Core terms (https://www.dublincore.org/specifications/dublin-core/dcmi-terms/) would be site-specific, but there are others that could be provided automatically by CKAN, such as `DCTERMS.created` and `DCTERMS.modified` for datasets and resources. Perhaps even `DCTERMS.publisher`, using the organisation. And `DCTERMS.type` will pretty much always be `text`.

It might also be worth supplying `AGLSTERMS.documentType`, eg "index" for list pages and packages, "dataset" for resource pages, "guidelines" for organisation 'About' pages and user profiles.

**Comment by loleg** [comment]
In regards to the various ways CKAN is used for publishing research data, this seems like a good idea. There is some relevant discussion in #1890 and an interesting project in the Frictionless Data community that could be useful: https://data-blog.gbif.org/post/frictionless-data-and-darwin-core/

**Comment by ThrawnCA** [comment]
We've added some DCTERMS metadata to our own theme, if an example would be helpful. Eg https://github.com/qld-gov-au/ckanext-data-qld/blob/0646186e79f97c74770913eb306fc8a699353256/ckanext/data_qld/templates/package/resource_read.html#L5
